sockeye/loss.py

```diff
@@ -27,40 +29,49 @@ class LossConfig(config.Config):
     """
     Loss configuration.
 
-    :param type: Loss name.
+    :param name: Loss name.
     :param vocab_size: Target vocab size.
     :param normalize: Whether to normalize loss value.
     :param smoothed_cross_entropy_alpha: Smoothing value for smoothed-cross-entropy loss.
     """
     def __init__(self,
-                 type: str,
+                 name: str,
```

Comment:
this is a backwards-incompatible change. You should bump the Sockeye version.

Reply:
Good catch.  When the rest of the PR is ready to go, I'll add a version increment commit.